net/unbound/files/README.md

```diff
@@ -0,0 +1,112 @@
+# Unbound Recursive DNS Server with UCI
+
+## Unbound Description
+Unbound is a validating, recursive, and caching DNS resolver. The C implementation of Unbound is developed and maintained by [NLnet Labs](https://www.unbound.net/). It is based on ideas and algorithms taken from a java prototype developed by Verisign labs, Nominet, Kirei and ep.net. Unbound is designed as a set of modular components, so that also DNSSEC (secure DNS) validation and stub-resolvers (that do not run as a server, but are linked into an application) are easily possible.
+
+## Package Overview
+Unbound may be useful on consumer grade embedded hardware. It is *intenteded* to be a recursive resolver only. [NLnet Labs NSD](https://www.nlnetlabs.nl/projects/nsd/) is *intenteded* for the authoritative task. This is different than ISC-Bind and its inclusive functions. As a consequence, Unbound configuration effort and memory consumption may be easier to control. A consumer could have their own recursive resolver, and remove potential issues from forwarding resolvers out side of their control.
+
+This package builds on Unbounds capabilities with OpenWrt UCI. Each Unbound option is not in UCI, but rather, UCI is used to simplify the combination of related options. Many of Unbounds native options are balanced and bundled to smaller user choices. Options include resources, DNSEC, access control, and some TTL tweaking. The UCI also provides an option to work at the raw "unbound.conf" level.
+
+## Work with dnsmasq
+The presented options will help Unbound and dnsmasq work together effeciently. Many want internal host identification in DNS. Alternative DHCP event scripts which write host records are difficult to formulate for Unbound, NSD, or Bind. In OpenWrt, dnsmsaq is the default DHCP and DNS stub resolver, and this will continue to serve internal DNS.
+
+Follow these instructions to make Unbound primary for handing DNS, and dnsmasq only serve local network DHCP-DNS. Set "option dnsmasq\_link\_dns" true and other related options as desired within Unbound. It will be necessary then to disable dnsmasq global lookups within "/etc/config/dhcp" and seciton "config dnsmasq." Set "option noresolv" true, remove "option resolvfile," and set "option port" to 1053 or 5353. Also for each "config dhcp \[*name*\], set "list dhcp_option option:dns-server,0.0.0.0." This last defends against internal combinations where dnsmasq may disable DHCP send options when it thinks it may not be performing DNS.
```

Comment:
`s/seciton/section/g`

Stopped reading at this point. Please run the document through a spell checker. :-)